CONTRIBUTING.rst

```diff
@@ -742,8 +495,56 @@ Resources & links
 
```

Comment:
Can't comment on the line here, but we the all-commits list is high volume so it's probably not recommended for most contributors to be subscribed to it. Even the dev list is optional for them.

Reply:
Added info on that